components/user/UserLink.js

```diff
@@ -31,12 +31,14 @@ export default function UserLink({
       target="_blank"
       rel="noopener noreferrer"
       onClick={clickLink}
-      className="rounded-full border-2 border-gray-200 hover:border-gray-500 hover:shadow-xl p-4 my-2 w-full content-start flex flex-row gap-4 items-center"
+      className="rounded-full border-2 border-gray-200 hover:shadow-xl p-4 my-2 w-full content-start flex flex-row gap-4 items-center user-link"
       style={{
-        color: colors[link.icon],
+        "--user-link-color": colors[link.icon]
```

Comment:
Could we use `border-color` directly here and not have a global style change?

Reply:
It depends.

How do you want to handle the border on hover?
Currently it goes from a light gray to dark gray.
The current PR code goes from light gray to icon color on hover.

I can make it a consistent icon color without using a css file.
I could also make a css file just for the UserLink component (hover state can't be modified in a style parameter).

Reply:
> I can make it a consistent icon color without using a css file.

Is this the simplest? I am thinking to do a small change, and then do further changes if required

I trust your judgement, I was just asking the question (I would prefer to avoid global styles if possible)

Reply:
I found a way to do the same thin using css variables and tailwind.

You can use css variables in [arbitrary properties](https://tailwindcss.com/docs/adding-custom-styles#resolving-ambiguities)

Reply:
oh nice, I didn't use that feature yet!

Reply:
I made the changes, let me know your thoughts.